.github/workflows/publish.yml

```diff
@@ -22,12 +22,18 @@ jobs:
 
     - name: Upgrade setuptools and install package
       run: |
-        python -m pip install --upgrade pip setuptools
-        python -m pip install -e .
+        python -m pip install build
 
     - name: Build source distribution
       run: python ./setup.py sdist
```

Comment:
Probably related to the last PR: does this work when we don't have setup.py any more?

Reply:
yes, `pyproject.toml` replaces both `setup.json/cfg` and `setup.py`. I wrote it in the PR description.

Reply:
oh, wait, you are right. I will fix this.

Reply:
Interesting thing to read: https://stackoverflow.com/questions/58753970/how-to-build-a-source-distribution-without-using-setup-py-file